CHANGELOG.md

```diff
@@ -1,5 +1,107 @@
 # Changelog
 
+## 4.7.0 (TBD)
+
+* Update bugsnag-android to v4.22.2:
+
+  * This release adds a compile-time dependency on the Kotlin standard library. This should not affect
+  the use of any API supplied by bugsnag-unity.
+
+  * Modularise bugsnag-android into Core, NDK, and ANR artifacts
+    [#522](https://github.com/bugsnag/bugsnag-android/pull/522)
+
+  * Add `detectNdkCrashes` configuration option to toggle whether C/C++ crashes
+    are detected
+    [#491](https://github.com/bugsnag/bugsnag-android/pull/491)
+
+  * Use NetworkCallback to monitor connectivity changes on newer API levels
+  [#501](https://github.com/bugsnag/bugsnag-android/pull/501)
+
+  * Migrate dependencies to androidx
+    [#554](https://github.com/bugsnag/bugsnag-android/pull/554)
+
+  * Report internal SDK errors to bugsnag
+    [#570](https://github.com/bugsnag/bugsnag-android/pull/570)
```

Comment:
Can you combine all internal reporting PR's into one summary? Users won't have experienced any of the incremental changes so they don't all need to be called-out.